### PR TITLE
Feature capstone project 1

### PR DESCRIPTION
**Pull request by kit0-0**
**This pull request includes the following changes to the Capstone project:**

- [ ]  Add Home page (Mobile/Desktop).
- [ ]  Add About page (Mobile/Desktop).
- [ ] Added CSS for Home and About pages
- [ ] Implemented dynamic elements on the Home page ( speaker and Program section).

These changes greatly improve the overall user experience and functionality of the Capstone project.

### CODE REVIEW

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @kit0-0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Very nice design :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
- Professional README :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kit0-0** [comment]
# New additions:

- Changed the background of the mobile menubar.
- Updated the heading on the home page specifically for the mobile version.
- Added italic style to the "Featured Speaker" element.
- Applied italic style and increased the font size to the "speaker-header," "main section heading," and "main-sec-heading-about" elements. Additionally, made the main section heading bold for better emphasis.
- Unfortunately, a suitable background picture for the main section heading could not be found.
- Resized the past projects section.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,846 @@
...
+}
+
+.footer-about-text {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .about-container {
+    background: url(./images/about_bg_01.png);
+  }
```
Comment (Zuhaib042) [inline style.css:739]: - [ ] As suggested by the previous reviewer, consider adding `background-color: #f7f7f7`to .about-container on all screen sizes to have light gray background-color in the about section.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+            </p>
+            <p class="contact-us">
+                Do you have other questions about the CC Global Summit 2015! </p>
+            <a href="#" class="email-id">ccsummit2015@cccanada.org </a>
+        </div>
+    </main>
+    <section class="section-2">
+        <h2 class="about-h2">Creative city Global Summit 2015 Logo</h2>
```
Comment (Zuhaib042) [inline about.html:52]: - [OPTIONAL] Consider giving this element more `margin-top` to make it similar to the original design.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            Through the logo contest, which was held from June 8th to July 7th, the winning design was determined
+            through online voting and evaluation by judges.
+            The logo of CC Global Summit 2015, which was optimized for use on the homepage and promotional materials
+            after consultation with the winning designer, is as follows.
+        </p>
+        <img src="./images/about_logo.png" alt="logo-2" class="about-logo">
+    </section>
+
+    <section class="section-3">
+        <h2 class="about-h2">View Past Global Summits</h2>
```
Comment (Zuhaib042) [inline about.html:64]: - [OPTIONAL] Consider giving this element more `margin-top` to make it similar to the original design.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <section class="speaker">
+        <h2 class="main-section-h1 ">Featured Speakers</h2>
+        <div class="orange-underline"></div>
+        <div class="speaker-container" id="container">
+        </div>
+        <button type="button" class="more-btn">More <img src="./images/arrow_down.png" alt="arrow_down"></button>
+
+    </section>
+    <!-- sponsers -->
+    <section class="partner hidden">
```
Comment (Zuhaib042) [inline index.html:75]: - [OPTIONAL] Kindly consider adding this background-color `#414246` to this section, and then `gray` color to the logos to make it similar to the behance design.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+        <div class="past-projects">
+            <div class="project-1 project">
+            </div>
+            <div class="project-2 project">
+            </div>
+        </div>
+    </section>
+
+    <!-- sponsers -->
+    <section class="partner about-partner">
```
Comment (Zuhaib042) [inline about.html:79]: - [OPTIONAL] Kindly consider adding this background-color #414246 to this section, and then gray color to the logos to make it similar to the behance design.

**Comment by kit0-0** [comment]
# New additions:
- Added background-color: #f7f7f7 to the .about-container class.
- Applied margin-top to section 2 and section 3 for proper spacing.
- Set the background-color to #414246 for the partner section.
- Modified the logo colors to have a consistent color throughout.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**